D:\applications\Alpha-Q-ai>python downloadqmoiai.py
Checking all download links for reachability...
BROKEN: Qbrowser (skv) [win] => https://downloads.qmoi.app/qbrowser/windows.exe (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qbrowser/windows.exe (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D81F38C0>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.523634] Broken download link | {"app": "Qbrowser (skv)", "platform": "win", "url": "https://downloads.qmoi.app/qbrowser/windows.exe", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qbrowser/windows.exe (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D81F38C0>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: Qbrowser (skv) [mac] => https://downloads.qmoi.app/qbrowser/mac.dmg (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qbrowser/mac.dmg (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D8314050>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.526085] Broken download link | {"app": "Qbrowser (skv)", "platform": "mac", "url": "https://downloads.qmoi.app/qbrowser/mac.dmg", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qbrowser/mac.dmg (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D8314050>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: Qbrowser (skv) [android] => https://downloads.qmoi.app/qbrowser/android.apk (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qbrowser/android.apk (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D8314A50>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.528152] Broken download link | {"app": "Qbrowser (skv)", "platform": "android", "url": "https://downloads.qmoi.app/qbrowser/android.apk", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qbrowser/android.apk (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D8314A50>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: Qbrowser (skv) [ios] => https://downloads.qmoi.app/qbrowser/ios.ipa (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qbrowser/ios.ipa (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D81C3C50>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.565475] Broken download link | {"app": "Qbrowser (skv)", "platform": "ios", "url": "https://downloads.qmoi.app/qbrowser/ios.ipa", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qbrowser/ios.ipa (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D81C3C50>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: Qbrowser (skv) [linux] => https://downloads.qmoi.app/qbrowser/linux.appimage (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qbrowser/linux.appimage (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D81C3D90>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.567528] Broken download link | {"app": "Qbrowser (skv)", "platform": "linux", "url": "https://downloads.qmoi.app/qbrowser/linux.appimage", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qbrowser/linux.appimage (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D81C3D90>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QFileManager (skv) [win] => https://downloads.qmoi.app/qfilemanager/windows.exe (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qfilemanager/windows.exe (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D81C3110>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.568994] Broken download link | {"app": "QFileManager (skv)", "platform": "win", "url": "https://downloads.qmoi.app/qfilemanager/windows.exe", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qfilemanager/windows.exe (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D81C3110>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QFileManager (skv) [mac] => https://downloads.qmoi.app/qfilemanager/mac.dmg (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qfilemanager/mac.dmg (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D81C3610>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.570402] Broken download link | {"app": "QFileManager (skv)", "platform": "mac", "url": "https://downloads.qmoi.app/qfilemanager/mac.dmg", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qfilemanager/mac.dmg (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D81C3610>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QFileManager (skv) [android] => https://downloads.qmoi.app/qfilemanager/android.apk (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qfilemanager/android.apk (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D81C2D50>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.571943] Broken download link | {"app": "QFileManager (skv)", "platform": "android", "url": "https://downloads.qmoi.app/qfilemanager/android.apk", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qfilemanager/android.apk (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D81C2D50>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QFileManager (skv) [ios] => https://downloads.qmoi.app/qfilemanager/ios.ipa (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qfilemanager/ios.ipa (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D81C2350>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.573402] Broken download link | {"app": "QFileManager (skv)", "platform": "ios", "url": "https://downloads.qmoi.app/qfilemanager/ios.ipa", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qfilemanager/ios.ipa (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D81C2350>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QFileManager (skv) [linux] => https://downloads.qmoi.app/qfilemanager/linux.appimage (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qfilemanager/linux.appimage (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D81C2850>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.574834] Broken download link | {"app": "QFileManager (skv)", "platform": "linux", "url": "https://downloads.qmoi.app/qfilemanager/linux.appimage", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qfilemanager/linux.appimage (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D81C2850>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QClock (skv) [win] => https://downloads.qmoi.app/qclock/windows.exe (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qclock/windows.exe (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D83142D0>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.576426] Broken download link | {"app": "QClock (skv)", "platform": "win", "url": "https://downloads.qmoi.app/qclock/windows.exe", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qclock/windows.exe (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D83142D0>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QClock (skv) [mac] => https://downloads.qmoi.app/qclock/mac.dmg (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qclock/mac.dmg (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D8314CD0>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.577957] Broken download link | {"app": "QClock (skv)", "platform": "mac", "url": "https://downloads.qmoi.app/qclock/mac.dmg", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qclock/mac.dmg (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D8314CD0>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QClock (skv) [android] => https://downloads.qmoi.app/qclock/android.apk (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qclock/android.apk (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D8315090>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.579398] Broken download link | {"app": "QClock (skv)", "platform": "android", "url": "https://downloads.qmoi.app/qclock/android.apk", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qclock/android.apk (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D8315090>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QClock (skv) [ios] => https://downloads.qmoi.app/qclock/ios.ipa (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qclock/ios.ipa (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D8315450>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.580883] Broken download link | {"app": "QClock (skv)", "platform": "ios", "url": "https://downloads.qmoi.app/qclock/ios.ipa", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qclock/ios.ipa (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D8315450>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QClock (skv) [linux] => https://downloads.qmoi.app/qclock/linux.appimage (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qclock/linux.appimage (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D8315810>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.582291] Broken download link | {"app": "QClock (skv)", "platform": "linux", "url": "https://downloads.qmoi.app/qclock/linux.appimage", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qclock/linux.appimage (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D8315810>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QMap (skv) [win] => https://downloads.qmoi.app/qmap/windows.exe (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qmap/windows.exe (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D8315BD0>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.583750] Broken download link | {"app": "QMap (skv)", "platform": "win", "url": "https://downloads.qmoi.app/qmap/windows.exe", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qmap/windows.exe (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D8315BD0>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QMap (skv) [mac] => https://downloads.qmoi.app/qmap/mac.dmg (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qmap/mac.dmg (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D8315F90>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.585240] Broken download link | {"app": "QMap (skv)", "platform": "mac", "url": "https://downloads.qmoi.app/qmap/mac.dmg", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qmap/mac.dmg (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D8315F90>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QMap (skv) [android] => https://downloads.qmoi.app/qmap/android.apk (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qmap/android.apk (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D8316210>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.586686] Broken download link | {"app": "QMap (skv)", "platform": "android", "url": "https://downloads.qmoi.app/qmap/android.apk", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qmap/android.apk (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D8316210>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QMap (skv) [ios] => https://downloads.qmoi.app/qmap/ios.ipa (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qmap/ios.ipa (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D81C2210>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.588862] Broken download link | {"app": "QMap (skv)", "platform": "ios", "url": "https://downloads.qmoi.app/qmap/ios.ipa", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qmap/ios.ipa (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D81C2210>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QMap (skv) [linux] => https://downloads.qmoi.app/qmap/linux.appimage (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qmap/linux.appimage (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D81C2990>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.590365] Broken download link | {"app": "QMap (skv)", "platform": "linux", "url": "https://downloads.qmoi.app/qmap/linux.appimage", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qmap/linux.appimage (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D81C2990>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QSearch (skv) [win] => https://downloads.qmoi.app/qsearch/windows.exe (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qsearch/windows.exe (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D81C2E90>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.591757] Broken download link | {"app": "QSearch (skv)", "platform": "win", "url": "https://downloads.qmoi.app/qsearch/windows.exe", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qsearch/windows.exe (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D81C2E90>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QSearch (skv) [mac] => https://downloads.qmoi.app/qsearch/mac.dmg (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qsearch/mac.dmg (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D81C2AD0>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.593174] Broken download link | {"app": "QSearch (skv)", "platform": "mac", "url": "https://downloads.qmoi.app/qsearch/mac.dmg", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qsearch/mac.dmg (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D81C2AD0>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QSearch (skv) [android] => https://downloads.qmoi.app/qsearch/android.apk (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qsearch/android.apk (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D81C2710>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.594554] Broken download link | {"app": "QSearch (skv)", "platform": "android", "url": "https://downloads.qmoi.app/qsearch/android.apk", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qsearch/android.apk (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D81C2710>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QSearch (skv) [ios] => https://downloads.qmoi.app/qsearch/ios.ipa (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qsearch/ios.ipa (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D81C3750>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.595911] Broken download link | {"app": "QSearch (skv)", "platform": "ios", "url": "https://downloads.qmoi.app/qsearch/ios.ipa", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qsearch/ios.ipa (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D81C3750>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QSearch (skv) [linux] => https://downloads.qmoi.app/qsearch/linux.appimage (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qsearch/linux.appimage (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D81C39D0>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.597294] Broken download link | {"app": "QSearch (skv)", "platform": "linux", "url": "https://downloads.qmoi.app/qsearch/linux.appimage", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qsearch/linux.appimage (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D81C39D0>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QAutoDev (skv) [win] => https://downloads.qmoi.app/qautodev/windows.exe (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qautodev/windows.exe (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D83165D0>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.599041] Broken download link | {"app": "QAutoDev (skv)", "platform": "win", "url": "https://downloads.qmoi.app/qautodev/windows.exe", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qautodev/windows.exe (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D83165D0>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QAutoDev (skv) [mac] => https://downloads.qmoi.app/qautodev/mac.dmg (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qautodev/mac.dmg (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D8316350>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.600753] Broken download link | {"app": "QAutoDev (skv)", "platform": "mac", "url": "https://downloads.qmoi.app/qautodev/mac.dmg", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qautodev/mac.dmg (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D8316350>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QAutoDev (skv) [android] => https://downloads.qmoi.app/qautodev/android.apk (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qautodev/android.apk (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D8315E50>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.602185] Broken download link | {"app": "QAutoDev (skv)", "platform": "android", "url": "https://downloads.qmoi.app/qautodev/android.apk", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qautodev/android.apk (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D8315E50>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QAutoDev (skv) [ios] => https://downloads.qmoi.app/qautodev/ios.ipa (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qautodev/ios.ipa (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D8315A90>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.603605] Broken download link | {"app": "QAutoDev (skv)", "platform": "ios", "url": "https://downloads.qmoi.app/qautodev/ios.ipa", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qautodev/ios.ipa (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D8315A90>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
BROKEN: QAutoDev (skv) [linux] => https://downloads.qmoi.app/qautodev/linux.appimage (error: HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qautodev/linux.appimage (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002E5D83156D0>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)")))
[2025-07-22T02:28:05.604988] Broken download link | {"app": "QAutoDev (skv)", "platform": "linux", "url": "https://downloads.qmoi.app/qautodev/linux.appimage", "error": "HTTPSConnectionPool(host='downloads.qmoi.app', port=443): Max retries exceeded with url: /qautodev/linux.appimage (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000002E5D83156D0>: Failed to resolve 'downloads.qmoi.app' ([Errno 11001] getaddrinfo failed)\"))"}
